trajectory.py

```diff
@@ -171,11 +188,13 @@ def _compute_distance(self, row):
         return dist_meters  
     
     def get_length(self):
+        """Return float of length of Trajectory object in measurement unit of CRS."""
```

Comment:
This description is not completely correct since get_length() calls _compute_distance() which returns distance in CRS units, except for trajectories where .is_latlon() is true in which case it returns meters.

Reply:
Made suggested change here